src/components/Comment/Comment.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from 'react';
+import noop from 'lodash/noop';
+
+import CommentComponent from '../../../third-party/components/Comment';
+
+import { PLACEHOLDER_USER } from '../../constants';
+
+type Props = {
+    id: string,
+    message: string,
+    createdAt: string,
+    createdBy?: User,
+    modifiedAt?: string,
+    permissions: AnnotationPermissions,
+    onDelete: Function,
+    isPending: boolean,
+    error?: ActionItemError
+};
+
+class Comment extends React.PureComponent<Props> {
+    static defaultProps = {
+        isPending: false,
+        permissions: {
+            can_delete: false,
+            can_edit: false
+        },
+        onDelete: noop
+    };
+
+    render() {
+        const { id, isPending, error, createdAt, modifiedAt, message, onDelete, permissions, createdBy } = this.props;
+        const createdByUser = createdBy && createdBy.name ? createdBy : PLACEHOLDER_USER;
```

Comment:
it looks like the CommentComponent itself checks the created_by user, can we just pass it through instead of this check then?

Reply:
Made it optional here and in CommentComponent